### DISCUSSION TITLE
Add more time intervals (e.g. week, month)

### DISCUSSION THREAD

**Discussion by helgehatt** [ghostfolio/ghostfolio#2568] [Ideas]
I can't believe I'm the only one that want something in between "Today" and "YTD", but I couldn't find any discussions or issues on it.

All other financial overviews such as yahoo and google include time intervals of 5D, 1M, 3M and 6M in addition to the existing ones.

Any reason why these are omitted?

**Comment by helgehatt** [comment]
One more thing, "Today" should in my opinion be replaced by "1D", and be the most recent **open** stock exchange day.

**Comment by amoerie** [comment]
You're not alone, I missed this as well.

I created a PR for this here: https://github.com/ghostfolio/ghostfolio/pull/2846
